src/stdlib/RandomMeasures.jl

```diff
@@ -82,7 +82,7 @@ function rand(rng::AbstractRNG, d::ChineseRestaurantProcess)
 end
 
 minimum(d::ChineseRestaurantProcess) = 1
-maximum(d::ChineseRestaurantProcess) = length(d.m) + 1
+maximum(d::ChineseRestaurantProcess) = any(i -> i == 0, d.m) ? length(d.m) : length(d.m)+1
```

Comment:
A bit more general (and simpler IMO) would be
```suggestion
maximum(d::ChineseRestaurantProcess) = any(iszero, d.m) ? length(d.m) : length(d.m) + 1
```

Reply:
Cool, thanks!